src/pages/home/report/ReportActionCompose/index.js

```diff
@@ -460,23 +467,42 @@ class ReportActionCompose extends React.Component {
         return _.size(this.props.reportActions) === 1;
     }
 
+    keydownListener(e) {
+        if (this.state.isFocused) { return; }
+
+        // if the key pressed is non-character keys like Enter, Shift, ... do not focus
+        if (e.key.length > 1) { return; }
+
+        // if we're typing on another input/text area, do not focus
+        if (e.target.nodeName === 'INPUT' || e.target.nodeName === 'TEXTAREA') { return; }
```

Comment:
nitpick: I don't have anything against retuning on the same line, but it isn't consistent with the codebase.

```suggestion
        if (e.target.nodeName === 'INPUT' || e.target.nodeName === 'TEXTAREA') { 
            return; 
        }
```